src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// React
+import React, { useState } from 'react';
+import { Routes, Route } from "react-router-dom"
+
+// Styling
+
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  NavbarText,
+} from 'reactstrap';
+
+// *********************
+// End styling
+// Begin content
+// *********************
+
+// Components
+import Footer from './components/Footer.js';
+import Header from './components/Header.js';
+
+// Pages
+import CatEdit from './pages/CatEdit.js';
+import CatIndex from './pages/CatIndex.js';
+import CatNew from './pages/CatNew.js';
+import CatShow from './pages/CatShow.js';
+import Home from './pages/Home.js';
+import NotFound from './pages/NotFound.js';
+
+// Imports mock cats
+import cats from './MockCats.js';
+
+// *********************
+// End imports
+// Begin App
+// *********************
+const App = (args) => { 
+  
+  const [isOpen, setIsOpen] = useState(true);
+  const toggle = () => setIsOpen(!isOpen);
+  
+  return (<>
+
+    <Header />
+
+    <Navbar {...args}>
+
+      <NavbarBrand href="/">Team Bengal</NavbarBrand>
+      <NavbarToggler onClick={toggle} />
+
+      <Collapse isOpen={isOpen} navbar>
+
+        <Nav navbar>
+          <NavItem><NavLink href="/catindex">See the cats</NavLink></NavItem>
+
+          <NavItem><NavLink href="/catshow">See a cat</NavLink></NavItem>
+
+          <NavItem><NavLink href="/catnew">Create a new cat</NavLink></NavItem>
+
+          <NavItem><NavLink href="/catedit">Edit a feline</NavLink></NavItem>
+
+          <NavItem><NavLink href="/*">woopsies</NavLink></NavItem>
+
+        </Nav>
+
+      </Collapse>
+
+    </Navbar>
```

Comment:
This NavBar block should be moved to a component to keep App.js clean, especially if you are using NavBar imported from Reactstrap.  Since this will likely show up in your header, you can move it there or create a separate component called navigation.